src/v/archival/ntp_archiver_service.cc

```diff
@@ -81,6 +81,30 @@ ntp_archiver::ntp_archiver(
       _start_term);
 }
 
+void ntp_archiver::run_sync_manifest_loop() {
+    vassert(
+      !_sync_manifest_loop_started,
+      "sync manifest loop for ntp {} already started",
+      _ntp);
+    _sync_manifest_loop_started = true;
+
+    // NOTE: not using ssx::spawn_with_gate_then here because we want to log
+    // inside the gate (so that _rtclog is guaranteed to be alive).
+    ssx::spawn_with_gate(_gate, [this] {
+        return sync_manifest_loop()
+          .handle_exception_type([](const ss::abort_requested_exception&) {})
+          .handle_exception_type([](const ss::sleep_aborted&) {})
+          .handle_exception_type([](const ss::gate_closed_exception&) {})
```

Comment:
if you handle the generic exception 

```
          .handle_exception([this](std::exception_ptr e) {
               vlog(_rtclog.error, "sync manifest loop error: {}", e);
           })
```

by catching it from `spawn_with_gate_then` instead of `spawn_with_gate` then you don't need to catch and ignore all of the other gate_closed_exception etc... exceptions because that will be done automatically

Reply:
oh, great. i didn't see this comment. thanks!